pkgs/development/python-modules/mammoth/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  cobble,
+  funk,
+  pytestCheckHook,
+  spur,
+  tempman,
+  gitUpdater,
+}:
+
+buildPythonPackage rec {
+  pname = "mammoth";
+  version = "1.8.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "mwilliamson";
+    repo = "python-mammoth";
+    tag = version;
+    hash = "sha256-mIvzEyKpy1j7tZti+QTXgsbqqZAhYs7N+p5sQFsLhBg=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail 'read("README")' '""'
+  '';
+
+  build-system = [ setuptools ];
+
+  dependencies = [ cobble ];
+
+  pythonImportsCheck = [ "mammoth" ];
+
+  nativeCheckInputs = [
+    funk
+    pytestCheckHook
+    spur
+    tempman
+  ];
+
+  disabledTestPaths = [
+    "tests/cli_tests.py"
```

Comment:
Can you maybe add a comment to say why this test file is disabled ?

Reply:
Nice catch there

Reply:
Fixed.